src/main/java/core/basesyntax/Info.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Info {
+    public void moreInfo();

```

Comment:
According to Java interface method declaration conventions, the 'public' modifier is redundant and can be omitted, as interface methods are implicitly public.